reporting/impl/src/main/java/org/jboss/windup/reporting/rules/generation/techreport/TechReportService.java

```diff
@@ -180,7 +180,7 @@ public TechStatsMatrix getTechStatsMap(ProjectModel onlyForApplication)
         for (TechnologyUsageStatisticsModel stat : statModels)
         {
             // A shortcut.
-            if (applicationProjects != null && !applicationProjects.contains(stat.getProjectModel()))
+            if (onlyForApplication != null && !stat.getProjectModel().getApplications().contains(onlyForApplication))
```

Comment:
The change seems fine and i think it could be good to delete the [`Set<ProjectModel> applicationProjects`](https://github.com/windup/windup/pull/1343/files?utf8=%E2%9C%93&diff=split#diff-6670edb730de259392f0270dbec38b2eR174) declaration above (sorry i can not comment on that line directly because it's "outside" of the changes in this commit)